ipaserver/install/httpinstance.py

```diff
@@ -577,3 +581,17 @@ def start_tracking_certificates(self):
         db = certs.CertDB(self.realm, nssdir=paths.HTTPD_ALIAS_DIR)
         db.track_server_cert(self.cert_nickname, self.principal,
                              db.passwd_fname, 'restart_httpd')
+
+    def request_service_keytab(self):
+        super(HTTPInstance, self).request_service_keytab()
+
+        if self.master_fqdn is not None:
+            service_dn = DN(('krbprincipalname', self.principal),
+                            ('cn', 'services'),
+                            ('cn', 'accounts'),
+                            self.suffix)
+
+            ldap_uri = ipaldap.get_ldap_uri(self.master_fqdn)
+            with ipaldap.LDAPClient(ldap_uri) as remote_ldap:
+                remote_ldap.gssapi_bind()
```

Comment:
Will the SASL/GSSAPI bind work also in domain level 0? I am not quite sure we request any host TGT during replica install in this case. We should verify this case so that we do not break migration paths from older masters.